src/types/index.tsx

```diff
@@ -1,11 +1,12 @@
-export type UserRole = "incharge" | "member" | "shift-incharge";
+export type UserRole = "incharge" | "member" | "shift_incharge";
 export type Shifts = "A" | "B" | "C" | "D";
 export type UserStatus = "active" | "inactive" | "pending";
+
 export type Member = {
   shift: string;
   id: number;
   name: string;
-  phone: string;
+  phoneNumber: string;
```

Comment:
@muhammad-bilal-ali-khokhar We should also consider the potential side effects of changing key entries.

Reply:
This looks fine. We did switch to `phoneNumber`, right? Good reminder though.